6_commands.ipynb

```diff
@@ -22,7 +22,7 @@
     "# Convenient commands\n",
     "<div class=\"alert alert-block alert-info\">\n",
     "    <b>Objective</b>\n",
-    "    <p>This notebook will collect convenient commands to help you.</p>\n",
+    "    <p>This notebook will collect glossary of utility commands to help you.</p>\n",
```

Comment:
"This notebook contains a glossary of utility commands to help you."
The verb "collect" needed to be replaced with "contains" and "a" needed to be added before "glossary".